src/coreclr/vm/exceptionhandling.cpp

```diff
@@ -1841,7 +1841,7 @@ CLRUnwindStatus ExceptionTracker::ProcessOSExceptionNotification(
     ExceptionTracker::InitializeCrawlFrame(&cfThisFrame, pThread, sf, &regdisp, pDispatcherContext, ControlPc, &uMethodStartPC, this);
 
 #ifndef ESTABLISHER_FRAME_ADDRESS_IS_CALLER_SP
-    uCallerSP = EECodeManager::GetCallerSp(cfThisFrame.pRD);
+    uCallerSP = cfThisFrame.GetCodeManager()->GetCallerSp(cfThisFrame.pRD);
 #else // !ESTABLISHER_FRAME_ADDRESS_IS_CALLER_SP
     uCallerSP = sf.SP;
```

Comment:
Do we need to do something in this other arm (looks like it's for ARM and ARM64 specifically) to support interp?

Reply:
This is the legacy EH code, this is not going to work with the interpreter anyways. Actually looking at the GetCallerSP usage, it seems that don't have a place where we would actually call it for the interpreter frame. I have made the change of `GetCallerSP` and `EnsureCallerContextIsValid` together, but maybe I could revert the `GetCallerSP`

Reply:
> maybe I could revert the GetCallerSP

Do you plan to do that?

Reply:
@kotas done in the last commit